__tests__/app.test.js

```diff
@@ -87,29 +94,6 @@ describe("GET /api/articles/:article_id", () => {
   });
 });
 
```

Comment:
Think about handling an invalid endpoint that doesn't exist eg /api/articles/banana